webapp/src/pages/Settings.tsx

```diff
@@ -249,6 +267,9 @@ const Settings: React.FC = () => {
         max: 1,
         step: 0.1,
       }}
+      InputProps={{
+        endAdornment: <InputAdornment position="end">ms</InputAdornment>,
```

Comment:
That should be `{UNITS[field]}`, right?
```suggestion
        endAdornment: <InputAdornment position="end">{UNITS[field]}</InputAdornment>,
```